packages/salesforcedx-vscode-lwc/src/commands/commandUtils.ts

```diff
@@ -6,12 +6,13 @@
  */
 
 import {
-  channelService,
+  ChannelService,
   notificationService
 } from '@salesforce/salesforcedx-utils-vscode/out/src/commands';
 import * as vscode from 'vscode';
 import { nls } from '../messages';
 
+const channelService = ChannelService.getInstance('lwc');
```

Comment:
I didn't find a better way of initializing the service per extension so we can have a unique channel name per extension. I could move this into its own module in the extension that does this initialization but other than that I think we'll have to do something like this per extension. Let me know if you have any suggestions

Reply:
I was thinking putting `window.createOutputChannel` in an `ChannelService.prototype.initialize` function, and put initialize in the extension activation function. That way we don't have to call `ChannelService.getInstance('lwc')` every time. (Also `lwc` should be an i18n string)

Reply:
Yeah that's cleaner, let me look at making those changes